packages/js/src/ui/context/AppearanceContext.tsx

```diff
@@ -25,6 +25,20 @@ export type Elements = {
   bellContainer?: ElementStyles;
   popoverContent?: ElementStyles;
   popoverTrigger?: ElementStyles;
+  settingsIconContainer?: ElementStyles;
+  inboxHeader?: ElementStyles;
+  inboxStatusDropdownTrigger?: ElementStyles;
+  inboxStatusDropdownContent?: ElementStyles;
+  inboxStatusDropdownItem?: ElementStyles;
+  inboxStatusDropdownItemLabel?: ElementStyles;
+  inboxStatusDropdownItemLeftIcon?: ElementStyles;
+  inboxStatusDropdownItemRightIcon?: ElementStyles;
+  inboxStatusTitle?: ElementStyles;
+  moreActionsDropdownTrigger?: ElementStyles;
+  moreActionsDropdownContent?: ElementStyles;
+  moreActionsDropdownItem?: ElementStyles;
+  moreActionsDropdownItemLabel?: ElementStyles;
+  moreActionsDropdownItemLeftIcon?: ElementStyles;
```

Comment:
We are going to be adding more appearance keys and I think we need to revise the naming here.
We should have a `dropdownTrigger`, `dropdownContent` etc. appearance keys for targeting all dropdowns
as well as `dropdownTrigger__moreActions` for targeting that specific dropdown. 
Does that make sense? Same for `inboxStatus`. 
In general, the element should come first in the name, like `dropdown`.